config.js

```diff
@@ -98,5 +98,17 @@ module.exports = {
       name: 'provider',
       url: 'https://provider.mainnet.oceanprotocol.com/spec'
     }
-  ]
+  ],
+  v3Versions: {
+    'ocean.js': 'v0.19.0',
+    'ocean.py': 'v0.8.5',
+    contracts: 'v0.6.9',
+    aquarius: 'v3.1.1',
```

Comment:
The latest v3  aqua version is: https://github.com/oceanprotocol/aquarius/releases/tag/v3.1.3